Assets/Scripts/Water/Editor/GeneratePlane.cs

```diff
@@ -0,0 +1,164 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.IO;
+
+namespace LowPolyWater
+{
```

Comment:
Opening curly brace must not be on new line: i.e. should be `namespace LowPolyWater {`